.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Description
+
+Add a brief summary of your PR here, between one sentence and two paragraphs
+
+## Additions
+
+- Detail API additions here if any
+- Do not list changes or removals
+
+## Removals
+
+- List API removals here if any
+
+## Modifications
+
+- List changes to existing structures and functions here if any
+
+## PR Checklist
+
+By placing an x in the boxes I certify that I have:
+
+- [ ] Ran `cargo test --all` locally if this modified any rs files.
+- [ ] Ran `cargo fmt --all` locally if this modified any rs files.
+- [ ] Updated the content of the book if this PR would make the book outdated.
+- [ ] Added a changelog entry if this will impact users.
```

Comment:
Even if it doesn't impact, we prefer when there is one, if the change is > 5 lines and is not documentation.